src/single-spa-vue.test.js

```diff
@@ -96,7 +124,6 @@ describe("single-spa-vue", () => {
 
   it(`passes appOptions straight through to Vue`, () => {
     const appOptions = {
-      el: document.createElement("div"),
```

Comment:
appOptions.el is supposed to be a string, not a dom element. So I removed this from this test and added a new test to cover when appOptions.el is passed in correctly as a string.